home/models.py

```diff
@@ -24,18 +25,51 @@
 )
 
 
+@register_setting
+class SiteSettings(BaseSetting):
+    content_variations_options = StreamField(
+        [
+            ("content_variations_options", blocks.CharBlock()),
+        ],
+        blank=True,
+        null=True,
+    )
+
+
 class MediaBlock(AbstractMediaChooserBlock):
     def render_basic(self, value, context=None):
         pass
 
 
+def get_choices():
+    if SiteSettings.objects.first():
+        return [
+            (choice, choice)
+            for choice in SiteSettings.objects.first().content_variations_options
+        ]
+    return []
+
+
+class VariationBlock(blocks.StructBlock):
+    variation_message = blocks.ChoiceBlock(
+        required=False,
+        choices=get_choices,
+        default=None,
+        help_text="Add variation fields to the site settings if you'd like to see them here",
```

Comment:
if we can, we could add this help text into a _() translation thing but happy to do that in maintenance